README.md

```diff
@@ -592,3 +593,103 @@ Looking for a specific version of QM?
 Search in the mirrors list below.
 
 [CentOS Automotive SIG - qm package - noarch](https://mirror.stream.centos.org/SIGs/9-stream/automotive/aarch64/packages-main/Packages/q/)
+
+## Changing QM Configuration
+
+### Overview
+
+To run QM on an immutable OSTree-based OS, we utilize systemd units with Podman Quadlet. Since modifying the original service file is not an option, we must use drop-in files to adjust the default configuration.
+
+For more information on how podman-systemd.unit works, refer to the manual by running:
+
+`man podman-systemd.unit`
+
+### Default Configuration File
+
+The default QM configuration drop-in file is located at:
+
+ /usr/share/containers/systemd/qm.container
+
+To override settings, create new drop-in files (ending in .conf) under the following directory `/etc/containers/systemd/qm.container.d/`. If the directory does not exist, create it first:
+
+ `mkdir -p /etc/containers/systemd/qm.container.d/`
+
+### Modifying the MemoryHigh Variable Example
```

Comment:
Consider that:

`### Example modifying QM configuration`

QM configuration modification is demonstrated through MemoryHigh variable.
Follow that demonstration with each configuration modification needed.